tests/math_tools/test_poisson_bracket.cpp

```diff
@@ -0,0 +1,94 @@
+#include <gtest/gtest.h>
+
+#include "../mapping/geometry_mapping_tests.hpp"
+
+#include "circular_to_cartesian.hpp"
+#include "combined_mapping.hpp"
+#include "cylindrical_to_cartesian.hpp"
+#include "gyrokinetic_poisson_bracket.hpp"
+#include "mesh_builder.hpp"
+#include "toroidal_to_cylindrical.hpp"
+
+
+TEST(GyrokineticPoissonBracket, Anticommutativity)
```

Comment:
More tests would be appreciated. Does anyone know any analytical cases where the result is known?

@pdonnel ?